waiter/test/waiter/scheduler/kubernetes_test.clj

```diff
@@ -91,7 +91,7 @@
      map->KubernetesScheduler)))
 
 (def dummy-service-description
-  {"backend-proto" "http"
+  {"backend-proto" "HTTP"
```

Comment:
Backend proto is always one of http and https. If this needs to be capitalized, that processing needs to happen elsewhere.

Reply:
I can't remember why I thought this was necessary... I was probably thinking that Waiter forces this to caps in production. Reverted to lower-case.